pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -124,6 +127,29 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			logger.Errorf("more than one CephCluster found in the namespace %q, choosing the first one %q", namespace, cephCluster.GetName())
```

Comment:
Is the intention to skip the rest of this for loop? If so we need a `continue`. How about making it a warning instead, and also clarifying that it only affects the crash collector